OAI_CONFIG_LIST_sample

```diff
@@ -7,14 +7,14 @@
     {
         "model": "gpt-4",
         "api_key": "<your Azure OpenAI API key here>",
-        "api_base": "<your Azure OpenAI API base here>",
+        "base_url": "<your Azure OpenAI API base here>",
         "api_type": "azure",
```

Comment:
Dont we need Deployment Name for Azure OpenAI Configs?

Reply:
"model" corresponds to the deployment name.